java/core/src/java/org/apache/orc/OrcConf.java

```diff
@@ -72,6 +72,21 @@ public enum OrcConf {
       "Define the compression strategy to use while writing data.\n" +
           "This changes the compression level of higher level compression\n" +
           "codec (like ZLIB)."),
+  COMPRESSION_ZSTD_IMPL("orc.compression.zstd.impl",
```

Comment:
This PR currently implements automatic fallback to aircompressor when `zstd-jni` is unavailable, and also implements the configuration of `orc.compression.zstd.impl=java` to force the use of aircompressor for writing.

However, there is currently no implementation of forcing the use of aircompressor for reading. Because it is a bit difficult to pass configuration items in the reader.

Reply:
For the reading part, let me take a look.